Tasks/VsTest/task.json

```diff
@@ -462,26 +462,29 @@
             "visibleRule": "rerunFailedTests = true"
         }
     ],
-    "sourceDefinitions": [
+    "dataSourceBindings": [
         {
             "target": "testPlan",
-            "endpoint": "/$(system.teamProject)/_apis/test/plans?filterActivePlans=true&api-version=3.0-preview.2",
-            "selector": "jsonpath:$.value[*].name",
-            "keySelector": "jsonpath:$.value[*].id",
-            "authKey": "tfs:teamfoundation"
+            "endpointId": "tfs:teamfoundation",
+            "endpointUrl": "{{endpoint.url}}/{{system.teamProject}}/_apis/test/plans?filterActivePlans=true&api-version=3.0-preview.2&$skip={{skip}}&$top=1000",
+            "resultSelector": "jsonpath:$.value[*]",
+            "resultTemplate": "{ \"Value\" : \"{{{id}}}\", \"DisplayValue\" : \"{{{name}}}\" }",
```

Comment:
100 by default should be good enough. Can you please share the new UI experience with this change?

Reply:
Are you sure 100 is enough? Because we are not stopping the loop. If there are many items, there will be too many calls.

Reply:
Also, from the experience side, there are no changes in UI. Just that user will feel a little delay while the results are fetched.